demo/Dockerfile

```diff
@@ -70,11 +71,10 @@ RUN  apt-get update \
 WORKDIR /root/agent
 
 # bring in the repo contents, minus .dockerignore files
-COPY . .
+COPY . cppagent
 
 ARG WITH_TESTS=false
 ARG WITH_TESTS_ARG=argument
-ARG SHARED=False
 
 # Build and optionally test
```

Comment:
IMHO we could make the two dockerfiles identical apart from OS differences e.g. deps installation).

```suggestion
# Build and optionally test. Unpack the dist to reduce overhead in the following release step.
```

Reply:
Indeed, that is a good idea. :+1: What language will you use? :thinking: If Bash or JavaScript/TypeScript, I could help you out if you want. :wink:

Reply:
I'm not sure if I will generate. Since we have the dockerhub account and the base image, I'll just use that as the foundation.

Reply:
I'll generate when I have more time.